stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2983,45 +2964,45 @@ struct _Native${Self}Storage<${TypeParametersDecl}> :
   }
 
   @discardableResult
-  internal mutating func updateValue(_ value: Value, forKey key: Key) -> Value? {
+  internal func updateValue(_ value: Value, forKey key: Key) -> Value? {
     _sanityCheckFailure(
       "don't call mutating methods on _Native${Self}Storage")
   }
 
   @discardableResult
-  internal mutating func insert(
+  internal func insert(
     _ value: Value, forKey key: Key
   ) -> (inserted: Bool, memberAfterInsert: Value) {
     _sanityCheckFailure(
       "don't call mutating methods on _Native${Self}Storage")
```

Comment:
Well, you're responsible for the PR now, so you should figure it out and/or ask him.